operator/operator.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: kudo.dev/v1beta1
 name: "cassandra"
-operatorVersion: "0.1.2"
+operatorVersion: "0.2.0"
```

Comment:
Let's revert the operator version bump. Version bumps should happen in the stable branches.